src/main/java/randoop/resource/ClassAnnotationScanner.java

```diff
@@ -0,0 +1,148 @@
+package randoop.resource;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+import org.objectweb.asm.AnnotationVisitor;
+import org.objectweb.asm.ClassVisitor;
+import org.objectweb.asm.MethodVisitor;
+import org.objectweb.asm.Opcodes;
+import org.objectweb.asm.TypePath;
+import org.plumelib.reflection.Signatures;
+
+/**
+ * ClassAnnotationScanner is a ClassVisitor that collects methods that are annotated with specific
+ * annotations.
+ */
+public class ClassAnnotationScanner extends ClassVisitor {
+  /** OpCode for ASM used throughout this class. */
+  private static final int ASM_OPCODE = Opcodes.ASM7;
```

Comment:
Is this an opcode or a version number?  I think it's the latter and should be documented as such.  It's only used once, so you could move its use to there.